docs/components/modeler/web-modeler/collaboration.md

```diff
@@ -30,8 +32,16 @@ Additionally, the **Owner** of the organization has special privileges to do adm
 
 #### Super-user mode
 
-Super-user mode is only available to the **Owner** of the organization. The main purpose of this mode is to assign collaborators
-to orphaned projects (which have no collaborators). Ordinarily, these projects would not be accessible or visible to any users.
+:::note
+Super-user mode is not yet available in Web Modeler Self-Managed.
+:::
```

Comment:
FYI: I thought it would make sense to mention it here because we already had Self-Managed customers asking for the super-user feature (see also https://github.com/camunda/issues/issues/276).